internal/core/metadata/uom/bootstrap.go

```diff
@@ -0,0 +1,47 @@
+//
+// Copyright (C) 2022 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package uom
+
+import (
+	"context"
+	"io/ioutil"
+	"sync"
+
+	bootstrapContainer "github.com/edgexfoundry/go-mod-bootstrap/v2/bootstrap/container"
+	"github.com/edgexfoundry/go-mod-bootstrap/v2/bootstrap/startup"
+	"github.com/edgexfoundry/go-mod-bootstrap/v2/di"
+	"github.com/pelletier/go-toml"
+
+	"github.com/edgexfoundry/edgex-go/internal/core/metadata/container"
+)
+
+func BootstrapHandler(_ context.Context, _ *sync.WaitGroup, _ startup.Timer, dic *di.Container) bool {
+	lc := bootstrapContainer.LoggingClientFrom(dic.Get)
+	config := container.ConfigurationFrom(dic.Get)
+	filepath := config.UoM.UoMFile
+
+	contents, err := ioutil.ReadFile(filepath)
```

Comment:
Not backwards compatible with old V2 configuration which will not have this field set. Should check for empty string and create empty `UnitsOfMeasureImpl` when it is empty. `Validation` will also be missing and default to false.